step.yml

```diff
@@ -37,6 +37,37 @@ inputs:
       The input value sets xcodebuild's `-destination` option.
     is_required: true
 
+# Test Configuration
```

Comment:
I wondered if we should add a step input category to the two new inputs.
For me, these two seem to have a similar priority to the test repetition-related inputs, which are under a category.

Reply:
Great idea. I have added a new category to both inputs.